_posts/2022-02-22-gleam-v0.19-released.md

```diff
@@ -0,0 +1,366 @@
+for finding all these locations and presenting them to the programmer to make a
+decide about what the logic should be.
+
+With Gleam v0.20.0 the compiler will return a helpful error message when it
+finds that a custom type variant has not been accounted for when pattern
+matching.
+
+```text
+error: Not exhaustive pattern match
+  ┌─ ./src/thingy.gleam:2:3
+  │
+2 │ ╭   case person {
+3 │ │     Teacher(..) -> "A teacher"
+4 │ │     Student(..) -> "A student"
+5 │ │   }
+  │ ╰───^
+
+This case expression does not match all possibilities. Each constructor
+must have a pattern that matches it or else it could crash.
+
+These values are not matched:
+
+  - Technician
+```
+
+Much better!
+
+Currently the compiler supported a limited form of this exhaustiveness checking
+for custom types. Support for other kinds of values and for nested values will
+come in a future release, along with generated code optimisations that use the
+pattern matching information to improve runtime code performance.
+
+Thank you to [Michał Łępicki](https://github.com/michallepicki) for this
+feature! As well as generally helping keeping the Gleam issue tracker squeaky
+clean and ship-shape.
+
+## GitHub support
+
+GitHub now recognises Gleam source code!
+
+Gleam source code will now be syntax highlighted on GitHub and Gleam will show
+in the list of languages for a given repo.
+
+<img class="no-shadow" style="border-bottom: 1px solid lightgrey" src="/images/news/gleam-v0.20-released/github-syntax-highlighting.png" alt="Some Gleam source code on GitHub with syntax highlighting">
+
+This isn't technically a Gleam feature, but it's an exciting change that we've
+been looking forward to for several years so it's worth including.
+
+Thank you to everyone who has shared their Gleam code on GitHub. Without you
+this wouldn't have been possible!
+
+
+## And the rest
+
+Those are the big new things, but as always there's lots of little improvements.
+
+The diagnostic messages for type errors within Gleam's `try` expressions have
+been greatly improved by [Julien D](https://github.com/silicium14). Thank you
+Julien! This should make one of Gleam's less common features more approachable
+and easier to learn.
+
+The build tool now copies the `ebin` directory into place if one is provided by
+an Erlang dependency that uses the `rebar3` build tool. This is rare but is the
+case for latest versions of the popular `cowboy` package.
+
+Project licence values are now verified as being valid SPDX licences, part of a
+wider BEAM ecosystem goal to validate licences used within the Hex package
+manager. Thank you to [Jonathan Arnett](https://github.com/J3RN) for this
+contribution and for his thorough research into the state of play with BEAM
+package licencing.
+
+The source code formatter's output format has been improved with special cases
+to make some common patterns look better.
+
+And as always, we've fixed some bugs 🐜
```

Comment:
There's a second similar sentence when linking to the changelog a few lines below. Maybe that one could be moved here?